inference-engine/src/vpu/myriad_plugin/myriad_executable_network.cpp

```diff
@@ -186,6 +186,12 @@ InferenceEngine::Parameter ExecutableNetwork::GetMetric(const std::string &name)
 
 InferenceEngine::CNNNetwork ExecutableNetwork::GetExecGraphInfo() {
     auto perfInfo = _executor->getPerfTimeInfo(_graphDesc._graphHandle);
+    if (_graphDesc._name == "importedNetwork") {
+        auto lvl = _log->level();
+        _log->setLevel(LogLevel::Warning);
+        _log->warning("Can't create graph from an imported network");
+        _log->setLevel(lvl);
+    }
     return buildRuntimeGraph(_graphMetaData, perfInfo);
```

Comment:
It will return empty CNNNetwork, since `buildRuntimeGraph()` will return empty network, but may be there is a better way to do it by putting return inside the if statement? I haven't found a good way to do it, but I'm trying to do it